src/client/scripts/sharejsEditor.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+/*global RoomMethods:false*/
+Template.room.helpers({
+    config: function() {
+        return function(editor) {
+            editor.setTheme("ace/theme/pastel_on_dark");
+            editor.setShowPrintMargin(false);
+            var prevText;
+
+            prevText = RoomMethods.getKeynotes(Session.get("roomNumber"));
+            if (prevText && prevText.length != 0) {
+                prevText = prevText.split("\n");
+                var prevSavedText = "";
+
+                for (var i = 0; i < prevText.length; i++) {
+                    prevSavedText += prevText[i] + "\n";
+                }
```

Comment:
??? what is this doing.  It looks like it has a string, spits in into an array on '\n' then remakes the string from each array element concatenating '\n' to it.  How is prevSavedText different from the original prevText?